spot-contracts/contracts/BondIssuer.sol

```diff
@@ -50,10 +53,15 @@ contract BondIssuer is IBondIssuer, OwnableUpgradeable {
     ///      then the issue window opens at Friday 2AM GMT every week.
     uint256 public issueWindowOffsetSec;
 
-    /// @notice An enumerable list to keep track of bonds issued by this issuer.
-    /// @dev Bonds are only added and never removed, thus the last item will always point
-    ///      to the latest bond.
-    EnumerableSetUpgradeable.AddressSet private _issuedBonds;
```

Comment:
The issuer is not upgradable, we will deploy a new one. Its fine to modify the storage layout.